### PR TITLE
Add CODEOWNERS file to Uberon

### PR DESCRIPTION
**Pull request by matentzn**
This will make @anitacaron, @matentzn and @gouttegd guardians of the main branch.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Uberon code owners (Guardians of the main branch)
+# GitHub documentation about CODEOWNERS file - https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+*       @anitacaron
```
Comment (gouttegd) [inline .github/CODEOWNERS:4]: This makes @anitacaron being systematically asked to review _any_ PR that touches _any_ file!  (except the PRs that only touch YAML files or the Makefiles)

If she’s fine with that then I have no objection, but I am not sure anyone should be tasked with so much.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+*.yaml @matentzn
+*Makefile @gouttegd
+*uberon.Makefile @gouttegd
```
Comment (gouttegd) [inline .github/CODEOWNERS:7]: Just to be sure, you know that one file can have more than _one_ owner, right? I didn’t ask to be the _exclusive_ owner of the Makefile, just to be one of them.

**Review by gouttegd — DISMISSED** [review]
Approve, _if_ @anitacaron explicitly confirms that she’s OK with being the default reviewer for all PRs.

File: .github/CODEOWNERS (hunk 2 above)
Comment (matentzn) [reply .github/CODEOWNERS:7]:
```suggestion
*uberon.Makefile @gouttegd @matentzn @anitacaron
```

Like this?

Comment (gouttegd) [reply .github/CODEOWNERS:7]: Good with me.

File: .github/CODEOWNERS (hunk 1 above)
Comment (anitacaron) [reply .github/CODEOWNERS:4]: That's fine.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**